recipe/conda_build_config.yaml

```diff
@@ -18,20 +16,21 @@ cudatoolkit:
 cudnn:
   - 7.6
 blas_impl:
-  - mkl                        # [x86 or x86_64]
-openblas:
-  - 0.3.3
+  - mkl         # [x86]
+  - openblas    # [not x86]
 mkl:
-  - 2019
+  - 2020        # [linux]
+  - 2019        # [osx]
+openblas:
+  - 0.3.10
 numpy:
-  - 1.11
+  - 1.16
```

Comment:
this looks wrong.  please see how python/numpy are declared in aggregate's cbc.  using here 1.16 in general is for sure not ok, and might be also one issue why python 3.10 doesn't here ... as for python 3.10 numpy 1.21 is mandatory

Reply:
This comment is based on an older version of the PR and is not applicable anymore.